lalrpop-snap/src/normalize/prevalidate/mod.rs

```diff
@@ -75,20 +75,6 @@ impl<'grammar> Validator<'grammar> {
                             "extern and match definitions are mutually exclusive"
                         );
                     }
-
```

Comment:
You shouldn't need to change anything in  `lalrpop-snap`, it is only a "snapshot" of `lalrpop` used to compile used to compile `lalrpop` itself and is only updated occasionally by running `.snap.sh`

Reply:
Thanks for the comment.
I only changed tests in `lalrpop-snap` because they were running using `cargo test`, and failed.
I could have run the tests only on lalrpop, indeed, using `cargo test lalrpop`... Sorry for that.

Reply:
Hmm, as long as nothing in snap is changed I'd expect the tests to still pass regardless :/ Are you sure you didn't accidentally do this change in snap and then ran the tests? (I know I accidentally open a file in snap sometimes at least.)

Reply:
Indeed, I made a change accidentally.
I just removed the changes in `snap` ;-)
Thanks!